components/cftapps_cluster_lb_backend/main.tf

```diff
@@ -1,22 +1,28 @@
-locals {
-  env = (var.env == "aat") ? "stg" : "${(var.env == "perftest") ? "test" : "${var.env}"}"
+data "azurerm_log_analytics_workspace" "log_analytics" {
```

Comment:
this should use the module for looking it up:

https://github.com/hmcts/terraform-module-log-analytics-workspace-id/

Reply:
Updated and re-ran here - https://dev.azure.com/hmcts/CNP/_build/results?buildId=124354&view=results